### ISSUE TITLE
About the performance of roi pooling

### ISSUE THREAD

**Issue by szq0214** [jwyang/faster-rcnn.pytorch#63]
Hi @jwyang, thanks for the nice work!

I'm trying to reproduce the accuracy with this code.  I found the performance of roi pooling layer was a little poor (~63.3% with vgg16 pre-trained model). After replacing with the roi align layer, I obtained the similar accuracy as you reported. I'm not sure if something is incorrect in my experimental setting or in your roi pooling implementation since all your results are reported with roi align. I wonder whether you can provide the results of roi pooling in your experiments.

**Comment by jwyang** [comment]
@szq0214, actually, I run all three roi pooling methods and the performance should be very similar to each other. I will re-train a vgg model.

**Comment by jwyang** [comment]
@szq0214 I have tried on my side, it works well. have you successfully get the right mAP based on roi_pooling?
